source/hardware/controllers/pioneer_ddj_flx4.rst

```diff
@@ -286,6 +287,75 @@ No.       Control                                                          Funct
 5-8       :hwlabel:`SHIFT` + Pad           Select next FX for the related stem
 ========  ===============================  ======================================================
 
+.. _pioneer-ddj-flx4-pitch-shift-mode:
+
+Pitch Shift Mode
+^^^^^^^^^^^^^^^^
+
+This mode uses the "KEY SHIFT" mode of the controller.
+
+It allows to modify a song pitch by semitones. This pitch modification is reset when a track is loaded.
```

Comment:
```suggestion
It allows to modify a song pitch by semitones.
```

This behavior depends on a setting in the user preferences. You may want to link that setting instead